test/lint/lint-format-strings.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2020 The Bitcoin Core developers
```

Comment:
```suggestion
# Copyright (c) 2018-2022 The Bitcoin Core developers
```

Reply:
Thank you for the review ! I have updated the PR as per your suggestions.